src/CalcManager/UnitConverter.cpp

```diff
@@ -900,9 +901,9 @@ void UnitConverter::Calculate()
 /// Trims out any trailing zeros or decimals in the given input string
 /// </summary>
 /// <param name="input">wstring to trim</param>
-void UnitConverter::TrimString(wstring& returnString)
+void UnitConverter::TrimString(_Inout_ wstring& returnString)
 {
-    if (returnString.find(L'.') == m_returnDisplay.npos)
```

Comment:
Bug fixed!

Reply:
What was the bug?  [`npos`](https://en.cppreference.com/w/cpp/string/basic_string/npos) is a static member of `basic_string`.  My understanding is that the condition should be `returnString.find(L'.') == wstring::npos`.  You can write `m_returnDisplay.npos` or `returnString.npos`, but I believe it should be the same either way because they are referring to the same static member.  We should still prefer `wstring::npos` because it makes it clear that the member is static instead of the current form which implies non-static.

Reply:
The bug wasn't technical but logical, it was weird to use an instance we don't even use in this function to get access to npos.

All methods in this class get access to `wstring::npos` via instances, I was intrigued but I thought it was made on purpose for a reason I ignored. But it looks like it's the only class doing that, so I'm totally fine with using wstring:npos instead!